applications/nrf5340_audio/src/modules/audio_sync_timer_rtc.c

```diff
@@ -160,6 +161,29 @@ static int audio_sync_timer_init(void)
 		return ret;
 	}
 
+	/* Initialize capturing of current timestamps */
+	ret = nrfx_dppi_channel_alloc(&dppi_channel_curr_time_capture);
+	if (ret - NRFX_ERROR_BASE_NUM) {
+		LOG_ERR("nrfx DPPI channel alloc error (I2S frame start) - Return value: %d", ret);
+		return ret;
+	}
+
+	nrf_rtc_subscribe_set(audio_rtc_timer_instance.p_reg,
+			      AUDIO_RTC_TIMER_CURR_TIME_CAPTURE,
```

Comment:
Suggest: Is it more logical to rename these:
`AUDIO_RTC_TIMER_CURR_TIME_CAPTURE` -> `AUDIO_SYNC_TIMER_CURR_TIME_CAPTURE_RTC `and 
`AUDIO_SYNC_TIMER_CURR_TIME_CAPTURE` -> `AUDIO_SYNC_TIMER_CURR_TIME_CAPTURE_HFCLK` ? At least `SYNC` should be part of both names as that is what they do.

Reply:
updated